src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -62,6 +72,15 @@
      */
     public static final String MSG_KEY = "arrangement.members.before.inner";
 
+    /** List of class member tokens. */
+    private static final List<Integer> CLASS_MEMBER_TOKENS = Arrays.asList(
+            TokenTypes.VARIABLE_DEF,
```

Comment:
Once #8048 will be done, this tokens are good candidates for a helper method like `TokenUtil.isClassMember` or something like this.